api/apps/articles/model/index.coffee

```diff
@@ -5,7 +5,7 @@
 _ = require 'underscore'
 db = require '../../../lib/db'
 async = require 'async'
-{ validate, onPublish, generateSlugs, generateKeywords, sanitizeAndSave, mergeArticleAndAuthor } = require './save'
+{ validate, onPublish, generateSlugs, generateKeywords, generateArtworks, sanitizeAndSave, mergeArticleAndAuthor } = require './save'
```

Comment:
Would be good to break this on two lines you can just add a new line after one of the commas or do two `{} = require`